analyzer/src/main/kotlin/managers/SpdxDocumentFile.kt

```diff
@@ -186,7 +186,7 @@ internal fun SpdxPackage.getVcsInfo(): VcsInfo? {
  * [SpdxPackage], or null if there is no such reference.
  */
 private fun SpdxPackage.locateExternalReference(type: SpdxExternalReference.Type): String? =
-    externalRefs.find { it.referenceType == type.typeName && it.referenceCategory == type.category }?.referenceLocator
+    externalRefs.find { it.referenceType == type }?.referenceLocator
```

Comment:
Commit message: Any explanation why deserialization was not "safe" before but now is?

Reply:
I agree "type safety" was a bad choice of words. I've improved the commit message.